frontend/src/v5/validation/containerAndFerderationSchemes/validators.ts

```diff
@@ -16,10 +16,14 @@
  */
 
 import * as Yup from 'yup';
-import filesize from 'filesize';
-import { formatMessage } from '../services/intl';
+import { formatMessage } from '@/v5/services/intl';
 
-const containerName = Yup.string()
+export const numberField = Yup.number().typeError(formatMessage({
+	id: 'settings.surveyPoint.error.number',
```

Comment:
Only minor, but if this is to be used for any number field then maybe it'd be best to have a more generic format message id, although I can see at the moment the only use is in survey points

Reply:
fixed